include/aws/crt/Api.h

```diff
@@ -125,12 +125,46 @@ namespace Aws
             /// @private
             static const Io::IsTlsAlpnSupportedCallback &GetBYOCryptoIsTlsAlpnSupportedCallback();
 
+            /**
+             * Gets the static default EventLoopGroup, creating it if necessary.
+             *
+             * @return EventLoopGroup* A pointer to the static default EventLoopGroup
+             */
+            static Io::ClientBootstrap *GetOrCreateStaticDefaultClientBootstrap();
```

Comment:
trivial: also, copy/paste error in comments. says "EventLoopGroup" but this is "ClientBootstrap"

Reply:
trivial: Add comments about how this default will be used in places where the user doesn't explicitly pass a pointer blah blah. Comment about it's lifetime. Make it a nice informative summary of everything the user might want to know about "default XYZ", then have all places that use the "default XYZ" link back to this function, where the authorative.

Repeat these similar comments for each `GetOrCreateDefaultXYZ()` function. It's fine to be a bit repetitive in docs, because users don't read them top to bottom.